src/material-experimental/mdc-chips/chip.ts

```diff
@@ -257,6 +257,12 @@ export class MatChip extends _MatChipMixinBase implements AfterContentInit, Afte
     },
     hasLeadingIcon: () => { return !!this.leadingIcon; },
     setAttr: (name: string, value: string) => {
+      // MDC is currently using this method to set aria-checked on choice and filter chips,
+      // which in the MDC templates have role="checkbox" and role="radio" respectively.
+      // We have role="option" on those chips instead; as such, we do not want aria-checked.
+      if (attr === 'aria-checked') {
```

Comment:
I think this is supposed to be `name`.

Reply:
Whoops, bad copy paste from https://github.com/angular/components/pull/16528. Thank you!